.gitignore

```diff
@@ -3,3 +3,4 @@
 /obj/
 
 *.user
+#node_modules
```

Comment:
Is this change intentional? It doesn't seem so

Reply:
probably not, thanks!